pkg/cmd/server/origin/master.go

```diff
@@ -309,11 +355,17 @@ func (c *MasterConfig) Run(protected []APIInstaller, unprotected []APIInstaller)
 
 	handler = open
 
+	handler = indexAPIPaths(handler)
```

Comment:
Hmm so we were not adding CORS headers to all the console paths before when the console was served separately.  The difference is that if the console is being served with the CORS headers then when are claiming that the console's JS is safe for cross-origin sharing to any of the specified CORS origins (which maybe thats ok), vs. just the API's JSON responses being declared as safe.